docker/debian-latest-arm64-openssl-1.1.x/run.sh

```diff
@@ -5,7 +5,7 @@ export DEBUG="*"
 
 yarn install
 
-DOCKER_PLATFORM_ARCH="linux/amd64"
```

Comment:
The tests passes regardless of this change as we're using a base Docker image which is implicitly tagged for `linux/arm64`. However, let's keep things consistent.